src/doc/src/reference/registry-index.md

```diff
@@ -139,7 +139,6 @@ explaining the format of the entry.
             // https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html.
             "req": "^0.6",
             // Array of features (as strings) enabled for this dependency.
-            // May be omitted since Cargo 1.84.
```

Comment:
The doc might be a bit ambiguous. By looking at #14838 and #14839, this omitting means, if empty then you can omit this.

If a package has a features, then the field cannot be omitted.